vpr/src/route/rr_graph_storage.cpp

```diff
@@ -709,3 +709,68 @@ t_rr_graph_view t_rr_graph_storage::view() const {
         vtr::make_const_array_view_id(edge_dest_node_),
         vtr::make_const_array_view_id(edge_switch_));
 }
+
+void t_rr_graph_storage::reorder(const vtr::vector<RRNodeId, RRNodeId>& order,
+                                 const vtr::vector<RRNodeId, RRNodeId>& inverse_order) {
```

Comment:
I believe the inverse_order argument is not required.  Also, if it is required, why not just compute the inverse_order in the method instead of taking it as an argument?

Reply:
The `order` is computed as the input, but the `inverse_order` is also used outside this method, so passing in just one of them would require computing the other twice. I could overload the method, but it's only used in one place.